Atomic/trust.py

```diff
@@ -120,8 +123,13 @@ def delete(self):
             policy_file.truncate()
 
     def default(self):
-        with open(self.policy_filename, 'r+') as policy_file:
-            policy = json.load(policy_file)
+        mode = "r+" if os.path.exists(self.policy_filename) else "w+"
+        with open(self.policy_filename, mode) as policy_file:
+            if mode == "r+":
+                policy = json.load(policy_file)
+            else:
+                policy = { "default":[{}] }
```

Comment:
I think we need the default type. See https://github.com/projectatomic/skopeo/blob/master/default-policy.json

```
{ "default": [ { "type": "insecureAcceptAnything" }] }
```